src/pages/certification/components/AuditorRunTestForm/AuditorRunTestForm.tsx

```diff
@@ -68,8 +68,11 @@ const AuditorRunTestForm: React.FC<IAuditorRunTestForm> = ({
     } : undefined
   });
 
-  if (profile && profile.dapp) {
-    forceValidate = true
+  const handleRepoFieldBlur = (event: React.FocusEvent<HTMLInputElement | HTMLTextAreaElement>) => {
+    if (event.target.value !== form.getValues().repoURL) {
```

Comment:
We want the access to be re-checked even in certain case where the repoUrl is not updated. Just to ensure the verification is intact. 

Was there a specific reason, why this was limited to within this condition?

Reply:
With this condition the onblur event will trigger the verification only if the field value is changed
I can remove this and it'll be triggered even if the field value it's not changed

Reply:
I need to do another round of regression to ensure this works. Will let you know after that.

Reply:
I believe I've verified all cases, and couldn't find any issues. So we can keep this. :+1: